packages/prisma-generator-proto/src/generate/hooks.ts

```diff
@@ -65,6 +65,26 @@ export type ${model.name} = {
 export type ${model.name}FormValues = ${formValueType};`;
 }
 
+/**
+ * Generate array of objects where each object has
+ * the name of the key, and its type.
+ *
+ * Primary key is omitted
+ */
+function generateFieldsArray(model: DMMF.Model): string {
+  const idFieldName = findIdFieldName(model);
+  const editableFields = model.fields
+    .filter((field) => field.kind === "scalar" && field.name !== idFieldName)
```

Comment:
`generateClientType` uses very similar filter to this one. Let's refactor the filtering logic into its own function.